### DISCUSSION TITLE
Why are some entities are not showing up (like "Inverter Voltage L1")?

### DISCUSSION THREAD

**Discussion by SmaMa82** [wills106/homeassistant-solax-modbus#333] [General]
Hi,

I have a SolaX X1 Hybrid G4 with a three phase meter and a battery. I can't find some of the entities, like Inverter Voltage L1, Inverter Voltage L2, Inverter Voltage L3, Inverter Current L1, Inverter Current L2, Inverter Current L3. I found it in the Plugin file, so why is it not showing up? I have installed version 2023.02.7b3

Bye,
Christian

**Comment by infradom** [comment]
These parameters are only applicable to 3-phase systems (X3 models)

**Reply by SmaMa82** [reply]
That is not correct. My Wallbox can read them, also in "plugin_solax.py" you can see it:  allowedtypes = X3 | GEN3 | GEN4 | GEN | HYBRID,

**Reply by wills106** [reply]
@infradom is correct, if your Inverter is a single phase Inverter it's not configured to read any of the three phase registers.

In the SolaX cloud do you see the three phases or just the single phase?
According to the Modbus documentation from SolaX these aditional L2 & L3 registers don't exist in the single phase Inverter.

You can see how your Inverter is configured here:

https://github.com/wills106/homeassistant-solax-modbus/blob/09b323c561486608830ef2d0030fe4080a61eec8/custom_components/solax_modbus/plugin_solax.py#L4466-L4492
